Source/WebCore/rendering/RenderMarquee.cpp

```diff
@@ -77,7 +77,7 @@ RenderMarquee::~RenderMarquee() = default;
 
 int RenderMarquee::marqueeSpeed() const
 {
-    int result = m_layer->renderer().style().marqueeSpeed();
+    auto result = static_cast<int>(m_layer->renderer().style().marqueeSpeed().asMilliseconds());
```

Comment:
Is this the right kind of conversion? Flooring rather than rounding? Can it be out of `int` range?

Reply:
As per usual, it's the equivalent of the existing conversion, to keep behavior the same. Previously, this truncation happened when setting the value on RenderStyle. A better long term solution is probably to just keep it a double here.